### PR TITLE
apply pop up functionality

### PR DESCRIPTION
**Pull request by erezfree29**
applied pop functinality , also cancelled the silver background color main which I found not needed which you will see as a conflict

### CODE REVIEW

**Comment by eroiyo** [comment]
# ♻️ REQUIRED CHANGES ♻️ 

Hello Partner, you are doing a very good job with the project, but I guess there are some issues we have to address before the review of our code reviewer...

image

there is a overflow in the X cordinate because, the main section is still visible, i sugges to to you instead of change the hieght of the main section, change it's display propiety to none.

if you like the margin, i suggest to you to add it in other way

File: src/scripts/spawn-cards.js [hunk 1]
```diff
@@ -19,11 +19,20 @@ export const spawnCard = (movie, target) => {
...
   commentButton.classList.add('card-button');
 
+  commentButton.addEventListener('click', () => {
+    document.querySelector('.card-container').style.display = 'none';
+    document.querySelector('.mname').textContent = cardTitle.textContent;
+    document.querySelector('.movie_cover').src = image.src;
+    document.querySelector('.summary_text').innerHTML = movie.show.summary;
+    document.querySelector('.comments_container').style.display = 'block';
+    document.querySelector('main').style.height = '30%';
+  });
```
Comment (eroiyo) [inline src/scripts/spawn-cards.js:34]: ⚪ you can change it there, if you have some question you can reply me.

**Review by eroiyo — CHANGES_REQUESTED** [review]

File: src/scripts/spawn-cards.js (hunk 1 above)
Comment (erezfree29) [reply src/scripts/spawn-cards.js:34]: Yes, I am aware that this needs to be addressed at a later stage but I am waiting until we update the API with comments and will add that to the next pull request.

Comment (eroiyo) [reply src/scripts/spawn-cards.js:34]: Ok, sound fine, lets continue

**Pull request merged**